libp2p/transports/wstransport.nim

```diff
@@ -181,8 +202,6 @@ method start*(
 
   trace "Listening on", addresses = self.addrs
 
-  self.running = true
```

Comment:
when is `self.running = true` happening?

Reply:
[Here!](https://github.com/vacp2p/nim-libp2p/blob/5f256049abefbc688fc9c15752f5e6a4052203a4/libp2p/transports/transport.nim#L43) When we do [`await procCall Transport(self).start(addrs)`](https://github.com/vacp2p/nim-libp2p/pull/1535/files/d80a77749c946e9c4cd08354b1305f01eda338e7#diff-32742335c1fc84ce5a63289b969f76e9bfb25c3185d7567ad47a5144599763f4R154)